packages/eslint-config-react-js/index.js

```diff
@@ -22,6 +22,48 @@ module.exports = {
 		'react-hooks/exhaustive-deps': 'error',
 		'no-param-reassign': 'warn',
 		'consistent-return': 'warn',
+		'padding-line-between-statements': [
+			'error',
+			{
+				blankLine: "always",
+				prev: "*",
+				next: [
+					"return",
+					"if",
+					"switch",
+					"for",
+					"while",
+					"try",
+					"throw",
+					"export",
+				],
+			},
+			{
+				blankLine: "always",
+				prev: [
+					"const",
+					"let",
+					"var",
+					"import",
+				],
+				next: "*",
+			},
+			{
+				blankLine: "any",
+				prev: [
+					"const",
+					"let",
+					"var",
+					"import",
+				],
+				next: [
+					"const",
+					"let",
+					"var",
+					"import"
+				],
```

Comment:
this allows to override above configuration to have a space after these, so we do not end up with:

```
import ...

import ...

const ...

const ...

let ...
```

End result can be e.g.:
```
import ...
import ...

const ...
const ...
let ...
```